backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc

```diff
@@ -37,6 +37,8 @@ include::modules/oadp-self-signed-certificate.adoc[leveloffset=+2]
 include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]
 include::modules/oadp-installing-dpa-1-3.adoc[leveloffset=+1]
 include::modules/oadp-enabling-csi-dpa.adoc[leveloffset=+2]
+include::modules/oadp-about-disable-node-agent-dpa.adoc[leveloffset=+2] 
+For more details, see xref:../../../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-jobs[Running tasks in pods using jobs].
```

Comment:
```suggestion

```
This link belongs in the **additional resources** section. And then to point users to this section [(guidelines),](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) you can say:
>"For more information, see "Running tasks in pods using jobs"."

In the `oadp-about-disable-node-agent-dpa.adoc` module